README.md

```diff
@@ -93,7 +93,7 @@ df1 = pd.read_csv(d1_path)
 df2 = pd.read_csv(d2_path)
 
 # Instantiate matcher and run
-matcher = Coma(strategy="COMA_OPT")
+matcher = Coma()
```

Comment:
Add the use_instances=False here, it is a tutorial to show that both options exist, even though it is the default